modules/xmpp/recording.js

```diff
@@ -334,10 +334,15 @@ Recording.prototype.setRecording = function(...args) {
 
 /**
  * Starts/stops the recording.
- * @param token token for authentication
+ * @param {Object} options
+ * @param {string} options.token token for authentication
+ * @param {string} options.streamId the stream ID to be used with Jibri in
+ * the streaming mode.
  * @param statusChangeHandler {function} receives the new status as argument.
  */
-Recording.prototype.toggleRecording = function(options, statusChangeHandler) {
+Recording.prototype.toggleRecording = function(
+    options = { },
+    statusChangeHandler) {
```

Comment:
I see `statusChangeHandler` is used unconditionally. Do we really support calling this function without args, or is it just working out of luck?

Reply:
Sorry, it should say calling JitsiConference.toggleRecording with no args

Reply:
Then how are we calling it? `conference.toggleRecording(undefined, statusChangeHandler);` ?

Reply:
If so, LGTM.

Reply:
We're calling it like this:
https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L1696
The purpose of this PR is to allow to pass undefined options.